index.html

```diff
@@ -1,4 +1,6 @@
 <main>
 <p>its me, mario</p>
 <p>peaches, peaches, peaches, peaches. I looooooooveeee you</p>
+<p>toad</p>
+<p>shawn</p>
```

Comment:
can we add one more name here
```suggestion
<p>shawn</p>
<p>naima</p>
<p>kian</p>
```